### ISSUE TITLE
linkpred模块代码含义

### ISSUE THREAD

**Issue by discivigour** [ruizhang-ai/L0-SIGN_Detecting_Benefitial_Feature_Interactions_for_Recommender_Systems#6]
请问LinkPred类的forward函数中的torch.log2()函数和
s = torch.sigmoid(sum_log/self.temp)
s = s * (self.inter_max - self.inter_min) + self.inter_min
到底是干什么用的呢？

**Comment by ethanmock** [comment]
Hi,

It maps the predictions to the hard concrete distribution. You can see the description in the Appendix "Approximate L0 Regularization with Hard Concrete Distribution".
